.gitignore

```diff
@@ -2,3 +2,7 @@
 *.junit.xml
 *.log
 *.ser
+
+.idea
+vendor
+composer.lock
```

Comment:
Remove those because they are not necessary if the test setup is correct (ai-controller-frontend extension is located in ./ext/ai-controller-frontend/ directory of aimeos-core checkout)